mappings/net/minecraft/util/Identifier.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS kp net/minecraft/util/Identifier
-	FIELD a owner Ljava/lang/String;
+	FIELD a namespace Ljava/lang/String;
```

Comment:
I don't think it is. 

Also if you decide to change this the getter needs updating to accommodate the rename.